File: app/src/main/java/com/gmail/fattazzo/formula1world/news/reader/AutosportComNewsReader.java
package com.gmail.fattazzo.formula1world.news.reader;

import org.jsoup.nodes.Element;

/**
 * @author fattazzo
 *         <p/>
 *         date: 21/07/17
 */
public class AutosportComNewsReader extends RssNewsReader {

    @Override
    protected String getChannelImage(Element channelElement) {
        return "http://www.autosport.com/img/favicons/120x120.png";
    }
}

File: app/src/main/kotlin/com/gmail/fattazzo/formula1world/news/reader/AutosportComNewsReader.kt
package com.gmail.fattazzo.formula1world.news.reader

import org.jsoup.nodes.Element

/**
 * @author fattazzo
 *
 *
 * date: 21/07/17
 */
class AutosportComNewsReader : RssNewsReader() {

    override fun getChannelImage(channelElement: Element): String {
        return "http://www.autosport.com/img/favicons/120x120.png"
    }
}
